utils/generate_dotnet_channels.js

```diff
@@ -139,6 +139,9 @@ for (const [name, value] of Object.entries(protocol)) {
     mixins.set(name, value);
 }
 
+if (!process.argv[2])
+  throw new Error('.NET repository needs to be specified as an argument' + process.argv[2]);
```

Comment:
Let's provide an example, because it should point to some `src/Playwright` IIRC?